docs/readmes/proposals/p027_magma_outbound_roaming_support.md

```diff
@@ -0,0 +1,134 @@
+# Outbound Roaming
+
```

Comment:
Same comment as Inbound roaming proposal, add the reference doc here

Reply:
@Arsenii-Oganov please link [gdoc](https://docs.google.com/document/d/17OYrYw6PMxFEWi_Ubx8eD1Qt3UT2VbchR_FIT5ivxI0/edit#) here